recorder.c

```diff
@@ -524,6 +524,10 @@ static void recorder_dump_entry(recorder_info      *rec,
                 case 'z':
                 case 'q':
                 case 'v':
+#ifdef _WIN32
```

Comment:
Hmmm. I wonder if we want to have the #ifdef here.

Pro: goes to "unsupported mode" if not on Windows, so does not pass a I to underlying printf
Cons: requires ifdef in caller code.

I'll merge as is for now.

Reply:
Read the MS documentation. It looks like they also have a Z specifier. Do you want another pass and add that one too?

Reply:
Source https://docs.microsoft.com/en-us/cpp/c-runtime-library/format-specification-syntax-printf-and-wprintf-functions?view=vs-2017

Reply:
Never used 'Z', but is a type not a size modifier, not sure if would fit with the recorder.
I mean, how do you store in the ring a ANSI_STRING or UNICODE_STRING?
In the documentation they state

> Typically, the Z type character is used only in driver debugging functions that use a conversion specification, such as dbgPrint and kdPrint

so it seems not much useful for userspace programs and kernel space has its facilities.